modules.d/06dbus/module-setup.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# called by dracut
+check() {
+  require_binaries dbus-daemon || return 1
+
+  return 255
+}
+
+depends() {
+  echo systemd
+  return 0
+}
+
+install() {
+  local DBUS_SERVICE=/usr/lib/systemd/system/dbus.service
+  if [[ -e $DBUS_SERVICE ]]; then
+    if [[ -L $DBUS_SERVICE ]]; then
+      DBUS_SERVICE=$(readlink $DBUS_SERVICE)
+    fi
+  else
+    DBUS_SERVICE=/etc/systemd/system/dbus.service
+    if [[ -e $DBUS_SERVICE ]]; then
+      if [[ -L $DBUS_SERVICE ]]; then
+        DBUS_SERVICE=$(readlink $DBUS_SERVICE)
+      fi
+    else
+      echo "Could not find dbus.service";
+      exit 1
+    fi
+  fi
+
+  inst_multiple \
+    $DBUS_SERVICE \
+    /usr/lib/systemd/system/dbus.socket \
+    /usr/bin/dbus-daemon \
+    /usr/bin/dbus-send
+
+  dbus_system_services="
+        org.freedesktop.systemd1
+        org.freedesktop.timedate1
+        org.freedesktop.hostname1
+  "
+  inst_dir      /etc/dbus-1/system.d
+  inst_dir      /usr/share/dbus-1/services
+  inst_dir      /usr/share/dbus-1/system-services
+  inst_multiple /etc/dbus-1/system.conf
+  inst_multiple /usr/share/dbus-1/system.conf \
+                /usr/share/dbus-1/services/org.freedesktop.systemd1.service
+  for service in $dbus_system_services ; do
+      inst_multiple        /etc/dbus-1/system.d/${service}.conf \
```

Comment:
These are in `/usr/share/dbus-1/system.d` on Fedora.

Reply:
I noticed that in our latest Tumbleweed snapshot the location has also changed to be /usr/share/dbus-1/system.d, we probably need to use `inst_any` here?

Reply:
Yes, I guess so.

Reply:
I removed them. They do not seem to be required in the first place. If we find out we do, we can still re-add them with the new paths.